src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -156,6 +156,8 @@ public void should_parse_language_thai(string postTitle)
 
         [TestCase("Movie.Title.1994.Bulgarian.1080p.XviD-LOL")]
         [TestCase("Movie.Title.1994.BGAUDIO.1080p.XviD-LOL")]
+        [TestCase("Movie.Title.1994.BG.AUDIO.1080p.XviD-LOL")]
+        [TestCase("Movie.Title.1994.BG.ENG.RU.AUDIO.1080p.XviD-LOL")]
```

Comment:
`BG.ENG.RU` would be multi? Are we sure want to force this to bulgarian as this will be opening the door to why not call that english or russian then?

Reply:
Well, I've included this test just to be clear that the regex would mark it as Bulgarian. 

If you think it would be a problem for the other languages I'll try to find another way of doing it without affecting them.

I actually initially tried with the case sensitive regex which was picking also subbed releases, but it seems that was fixed, so maybe I should make some more tests 🤔

Thanks for the review!

Reply:
Nope, my fault, that was on Sonarr. I just transferred the idea here thinking the code is almost the same and missed the subs fix... I'll try doing it the proper way, thanks again :) 

Btw, someone should add that fix to Sonarr too!